Dfe.Data.SearchPrototype/Web/Tests/PageIntegrationTests/SearchPageTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using Dfe.Data.SearchPrototype.Web.Tests.PageObjectModel;
+using Dfe.Data.SearchPrototype.Web.Tests.Shared.Helpers;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Dfe.Data.SearchPrototype.Web.Tests.Integration
+{
+    public class SearchPageTests : IClassFixture<PageWebApplicationFactory>
+    {
+        private const string uri = "http://localhost:5000";
+        private readonly HttpClient _client; 
+        private readonly ITestOutputHelper _logger;
+        private readonly WebApplicationFactory<Program> _factory;
+
+        public SearchPageTests(PageWebApplicationFactory factory, ITestOutputHelper logger)
+        {
+            _factory = factory;
+            _client = factory.CreateClient(new WebApplicationFactoryClientOptions
+            {
+                AllowAutoRedirect = true
+            });
+            _logger = logger;
+        }
+
+        [Fact]
+        public async Task Search_Title_IsDisplayed()
+        {
+            var response = await _factory.CreateClient().GetAsync(uri);
+
+            var document = await HtmlHelpers.GetDocumentAsync(response);
+
+            document.GetElementText(SearchPage.Heading.Criteria).Should().Be("Search prototype");
+        }
+
+        [Fact]
+        public async Task Search_Establishment_IsDisplayed() 
+        {
+            var response = await _factory.CreateClient().GetAsync(uri);
+
+            var document = await HtmlHelpers.GetDocumentAsync(response);
+
+            document.GetElementText(SearchPage.SearchHeading.Criteria).Should().Be("Search");
+            
+            document.GetElementText(SearchPage.SearchSubHeading.Criteria).Should().Be("Search establishments and check their performance");
+            
```

Comment:
This is a great example of having to write the tests to fit the behaviour because we haven't defined what we expect. I'd say this behaviour is wrong - we're not comparing performance